content/components/ota/zephyr_mcumgr.md

```diff
@@ -0,0 +1,73 @@
+---
+description: "Instructions for setting up ESPHome's nRF52 platform to allow remote firmware updates of devices."
+title: "ESPHome nRF52 Firmware Updates"
+params:
+  seo:
+    description: Instructions for setting up ESPHome's nRF52 platform to allow remote firmware updates of devices.
+    image: system-update.svg
+---
+
+The nRF52 platform supports firmware updates via Over-the-Air (OTA) mechanisms. The **only supported bootloader** for OTA updates is **MCUboot**.
```

Comment:
Can we improve the documentation on what we need around MCUboot? Most of what I read suggests that we only need slot0 partition but I suspect we need at least two slots for MCUBoot? My nice!nano nRF52 currently has an Adarfruit boot loader and I'm not certain what is required.

Reply:
Best if you could open a PR with redaction of the doc. Push to this branch or just open new PR. I can link it to main PR. `nrf52` doc could be improved in few places but I have no much time to think about it.

In the nut shell you need:
1. flash firmware using SWD e.g. by stlink https://esphome.io/components/nrf52/#flashing-with-mcuboot
2. From that moment you can flash new firmware using serial, cdc serial or ble

By default it use slot0 and slot1. https://docs.nordicsemi.com/bundle/ncs-latest/page/mcuboot/readme-zephyr.html